src/api/types.js

```diff
@@ -9,11 +9,11 @@
  */
 
 /**
- * @typedef {Omit<FormConfiguration, 'id'>} FormConfigurationDocumentInput
- * @typedef {Omit<FormConfigurationDocumentInput, 'slug'>} FormConfigurationInput
- * @typedef {Request<{ Server: { db: import('mongodb').Db } }>} RequestDefaults
- * @typedef {RequestDefaults & Request<{ Params: { id: string } }>} RequestFormById
- * @typedef {RequestDefaults & Request<{ Payload: FormConfigurationInput }>} RequestCreateForm
+ * @typedef {Omit<FormConfiguration, 'id'>} FormConfigurationDocument
```

Comment:
Renamed this to ~`FormConfigurationDocumentInput`~ `FormConfigurationDocument`

It's used by all the MongoDB queries too which are kinda output not input